bindings/pyroot_experimental/pythonizations/CMakeLists.txt

```diff
@@ -92,7 +92,13 @@ foreach(val RANGE ${how_many_pythons})
 
   set(libname ROOTPythonizations${python_under_version_string})
 
-  ROOT_LINKER_LIBRARY(${libname} ${cpp_sources} DEPENDENCIES Core Tree cppyy${python_under_version_string} CMAKENOEXPORT)
+  add_library(${libname} SHARED ${cpp_sources})
```

Comment:
To be done: when removing `ROOT_LINKER_LIBRARY`, we need to add an installation command that such macro would add automatically. See for example `libcppyy`:

https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/cppyy/CPyCppyy/CMakeLists.txt#L101-L104